src/server/player_sao.cpp

```diff
@@ -525,6 +525,11 @@ ItemStack PlayerSAO::getWieldedItem(ItemStack *selected, ItemStack *hand) const
 	return m_player->getWieldedItem(selected, hand);
 }
 
+bool PlayerSAO::getOffhandWieldedItem(ItemStack *offhand, ItemStack *place, IItemDefManager *itemdef_manager, PointedThing pointed) const
+{
+	return m_player->getOffhandWieldedItem(offhand, place, itemdef_manager, pointed);
+}
```

Comment:
This function is probably not needed. You can use `player` in serverpackethandler.cpp to access this function directly.

Reply:
I added this function for symmetry to `PlayerSAO::getWieldedItem`. Is there a reason why getWieldedItem should exist but getOffhandWieldedItem shouldn't?

Reply:
`getWieldIndex`, `getWieldedItem` and `getOffhandWieldedItem` use already publicly exposed functions from `RemotePlayer`, which is used broadly in `serverpackethandler.cpp`.
The three functions only add another layer of abstraction, which is increases code bloat and maintenance efforts. Anyway. keep it as-is so that everything can be cleaned up in a future PR.

Reply:
By "keep it as-is", do you mean keeping the getOffhandWieldedItem function or keeping it's absence?

Reply:
I meant the former. Or just do whatever you want. This is not a change request.